scalafix-rules/src/main/scala/scalafix/internal/rule/ExplicitResultTypes.scala

```diff
@@ -19,10 +21,20 @@ final class ExplicitResultTypes(
 
   def this() = this(ExplicitResultTypesConfig.default, LazyValue.now(None))
 
-  private def supportedScalaVersion = Properties.versionNumberString
+  private def supportedScalaVersions: List[String] = {
```

Comment:
I am not completely satisfied of duplicating this logic. Is there a way to retrieve `supportedScalaVersionsForExplicitResultTypeRule` from builldInfo

Reply:
definitely! you should define a specific buildInfo for the rules project in build.sbt - look for stuff with the `buildInfo` prefix there to see how it works (keys + plugin activation)